helpers/example/components/AppStart.js

```diff
@@ -42,7 +42,6 @@ export function AppStart({ components, packageName, pkg, version }) {
 							`}
 						/>
 						<Sidebar components={components} brand={brand} setBrand={setBrand} />
-						<p>{version}</p>
```

Comment:
Why remove the version?

Reply:
It’s in there twice (already in the page heading), that one is breaking the page layout

Reply:
uhm not sure what you mean but the version is not duplicated anywhere I can find???

Reply:
<img width="1680" alt="Screen Shot 2020-05-19 at 10 37 25 am" src="https://user-images.githubusercontent.com/2049709/82272191-3711bf00-99bd-11ea-8500-0ae8423d18cf.png">

Reply:
got you